rootfs/rootfs/etc/vmware-tools/scripts/vmware/network

```diff
@@ -0,0 +1,596 @@
+#!/bin/sh
```

Comment:
I hate that we've fragmented this discussion :cry: -- https://github.com/boot2docker/boot2docker/pull/638#issuecomment-66576677: is there any way we could just keep a patch to the `open-vm-tools` version instead of a full copy?  Better yet would be to figure out a way to make the original work, but I'll settle for this if we can have some clear way of saying "this is the original and this is what had to be changed for boot2docker", especially so that we can update as necessary later and not forget what was broken and had to be fixed here in the interim.